lib/cjs/components/view/selected-items/selected-items.scss

```diff
@@ -63,7 +63,7 @@
     cursor: default;
     font-size: $oc-font-size-normal;
     font-weight: $oc-font-weight-normal;
-    margin: 0.1rem 0 0 2.4rem;
+    margin: 0 0 0 2.4rem;
```

Comment:
Did you tested selected items closed and opened in IE and Edge without that rule? It worked good in Chrome but not in IE